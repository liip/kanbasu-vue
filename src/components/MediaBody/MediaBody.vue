<template>
  <div class="media__body">
    <slot></slot>
  </div>
</template>

<script>
export default {
  name: 'MediaBody',
};
</script>
