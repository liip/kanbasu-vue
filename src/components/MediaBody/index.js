import MediaBody from './MediaBody.vue';

MediaBody.install = function(Vue) {
  Vue.component(MediaBody.name, MediaBody);
};

export default MediaBody;
