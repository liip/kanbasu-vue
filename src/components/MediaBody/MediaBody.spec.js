import { shallowMount } from '@vue/test-utils';
import MediaBody from './MediaBody.vue';

describe('MediaBody.vue', () => {
  it('renders', () => {
    const wrapper = shallowMount(MediaBody, {
      slots: {
        default: 'slot',
      },
    });
    expect(wrapper).toMatchSnapshot();
  });
});
