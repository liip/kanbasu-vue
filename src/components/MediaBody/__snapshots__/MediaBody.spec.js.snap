// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MediaBody.vue renders 1`] = `<div class="media__body">slot</div>`;
